packages/perseus/src/widgets/radio/__stories__/choice-icon.stories.jsx

```diff
@@ -1,161 +1,242 @@
+    correct: boolean,
+    showCorrectness: boolean,
+    product: "sat" | "library",
+    multipleSelect: boolean,
+    reviewMode: boolean,
+    previouslyAnswered: boolean,
+    transparentBackground: boolean,
+    primaryProductColor?: string,
 |};
 
-export default ({
-    title: "Perseus/Widgets/Radio/Choice Icon",
-}: Story);
-
-const defaultObject = {
-    crossedOut: false,
+const defaultProps = {
+    pos: 0,
     checked: false,
-    correct: true,
-    focused: false,
-    hovered: false,
-    pressed: false,
-    multipleSelect: false,
-    previouslyAnswered: false,
-    product: "library",
-    reviewMode: false,
-    showCorrectness: false,
-};
-
-// Note: the SAT icon is different from the general library
-// icon displayed on KA. Therefore props like pressed,
-// previouslyAnswered and focused do not apply here.
-// That said, the props of the component still want them as they are
-// currently defined, so we give them values anyway.
-const satObject = {
     crossedOut: false,
     focused: false,
     hovered: false,
     pressed: false,
-    checked: false,
-    correct: true,
+    correct: false,
+    showCorrectness: false,
+    product: "library",
     multipleSelect: false,
-    previouslyAnswered: false,
-    product: "sat",
     reviewMode: false,
-    showCorrectness: false,
+    previouslyAnswered: false,
+    transparentBackground: false,
 };
 
-export const Position0Sat = (args: StoryArgs): React.Node => {
-    return <ChoiceIcon {...satObject} pos={0} />;
+const defaultSATProps = {
+    ...defaultProps,
+    product: "sat",
 };
 
-export const Position1CrossedOutSat = (args: StoryArgs): React.Node => {
-    return <ChoiceIcon {...satObject} pos={1} crossedOut={true} />;
+export default {
+    title: "Perseus/Widgets/Radio/Choice Icon",
+    argTypes: {
+        pos: {
+            control: {type: "number", min: 0, max: 25, step: 1},
+        },
+        product: {
+            options: ["library", "sat"],
+            control: {type: "radio"},
+        },
+    },
+    args: defaultProps,
 };
 
-export const Position2SelectedSat = (args: StoryArgs): React.Node => {
-    return (
-        <ChoiceIcon {...satObject} pos={2} checked={true} crossedOut={false} />
-    );
+export const Interactive = (args: StoryArgs): React.Node => {
+    return <ChoiceIcon {...args} />;
 };
 
-export const Position2SelectedReviewModeSat = (args: StoryArgs): React.Node => {
+export const Default = (args: StoryArgs): React.Node => {
     return (
-        <ChoiceIcon
-            {...satObject}
-            pos={2}
-            checked={true}
-            crossedOut={false}
-            reviewMode={true}
-        />
+        <>
+            <p>Library</p>
+            <ChoiceIcon {...defaultProps} />
+            <ChoiceIcon {...defaultProps} multipleSelect={true} />
+            <p>SAT</p>
+            <ChoiceIcon {...defaultSATProps} />
+            <ChoiceIcon {...defaultSATProps} multipleSelect={true} />
+        </>
     );
 };
```

Comment:
**praise:** Nice!  I like having a story like this when there's lots of possible combinations.